packages/jaeger-ui-components/src/TraceTimelineViewer/SpanLinks.tsx

```diff
@@ -78,7 +78,7 @@ export const SpanLinksMenu = ({ links }: SpanLinksProps) => {
   const closeMenu = () => setIsMenuOpen(false);
 
   return (
-    <>
+    <div data-testid="span-row">
```

Comment:
if we need to add a testid here, it should probably be something like `SpanLinksMenu` instead of `span-row`

Reply:
ok I renamed it, but do you think that it could be done in a better way?